pyvex/lift/util/instr_helper.py

```diff
@@ -223,7 +223,7 @@ def constant(self, val, ty):
 
     def lookup_register(self, arch, reg):
         if isinstance(reg, int):
-            reg = arch.register_names[reg]
+            reg = arch.register_index[reg]
```

Comment:
What's `arch.register_index` supposed to be? I couldn't find any mention of it in archinfo.